wolfcrypt/src/aes.c

```diff
@@ -11863,7 +11863,7 @@ static WARN_UNUSED_RESULT int wc_AesFeedbackCFB1(
     }
 
     if (ret == 0) {
-        if (bit > 0 && bit < 7) {
+        if ((bit > 0 && bit < 7) || (bit == 0 && sz < 8)) {
```

Comment:
Reviewing this more, I don't think there is a case here where 'sz' does not equal 0 once it hits this check? Having broken out of the while loop (sz > 0) and 'ret' is 0. Think the bounds check on 'bit' might just need to be inclusive with 0 `if ((bit >= 0 && bit < 7)) {  `. Cases where 'bit' is 7 then out[0] is updated in the while loop and all other cases out[0] should be updated with the last value of cur after exiting the while loop.

Reply:
Just check bit < 7.
7 indicates no bits encrypted and not outputted.

Reply:
Done